discord/sticker.py

```diff
@@ -0,0 +1,128 @@
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+"""
+
+from .mixins import Hashable
+from .asset import Asset
+from .utils import snowflake_time
+from .enums import StickerType, try_enum
+
+class Sticker(Hashable):
+    """Represents a message sticker.
+
+    .. versionadded:: 2.0
+
+    .. container:: operations
+
+        .. describe:: str(x)
+
+            Returns the name of the sticker
+
+        .. describe:: x == y
+
+           Checks if the sticker is equal to another sticker
+
+        .. describe:: x != y
+
+           Checks if the sticker is not equal to another sticker
+
+    Attributes
+    ----------
+
+    name: :class:`str`
+        The sticker's name
+    id: :class:`int`
+        The id of the sticker
+    description: :class:`str`
+        The description of the sticker
+    pack_id: :class:`int`
+        the id of the sticker's pack
+    format: :class:`StickerType`
+        The format for the sticker's image
+    image: :class:`str`
+        The sticker's image
+    tags: List[:class:`str`]
+        A list of tags for the sticker
+    """
+    def __init__(self, state, data):
+        self._state = state
+        self.id = int(data["id"])
+        self.name = data["name"]
+        self.description = data["description"]
+        self.pack_id = int(data["pack_id"])
+        self.format = try_enum(StickerType, data["format_type"])
+        self.image = data["asset"]
+        self.tags = [tag.strip() for tag in data.get("tags", "").split(",")]
+        self.preview_asset = data.get("preview_asset")
+
+    def __repr__(self):
+        return "<{0.__class__.__name__} id={0.id} name='{0.name}'>".format(self)
+
+    def __str__(self):
+        return self.name
+
+    @property
+    def created_at(self):
+        """:class:`datetime.datetime`: Returns the stickers's creation time in UTC."""
+        return snowflake_time(self.id)
+
+    @property
+    def image_url(self):
+        """Returns an :class:`Asset` for the stickers image.
+
+        .. note::
+            This will return ``None`` if the format is ``StickerType.lottie``
+
+        Returns
+        -------
+        Optional[:class:`Asset`]
+            The resulting CDN asset.
+        """
+        return self.image_url_as()
+
+    def image_url_as(self, *, size=1024):
+        """Optionally returns an :class:`Asset` for the stickers image.
+
+        size must be a power of 2 between 16 and 4096.
+
+        .. note::
+            This will return ``None`` if the format is ``StickerType.lottie``.
+
+        Parameters
+        -----------
+        size: :class:`int`
+            The size of the image to display.
+
+        Raises
+        ------
+        InvalidArgument
+            invalid ``size``.
```

Comment:
```suggestion
            Invalid ``size``.
```
Since the others begin with a capital letter, I assume this one should too?